consensus/tendermint/config.go

```diff
@@ -73,7 +73,7 @@ func (tmintConfig *TendermintConfig) AssertTendermintDefaults(chainId, workDir,
 	tmintConfig.SetDefault("db_backend", "leveldb")
 	tmintConfig.SetDefault("db_dir", dataDir)
 	tmintConfig.SetDefault("log_level", "info")
-	tmintConfig.SetDefault("rpc_laddr", "0.0.0.0:46657")
```

Comment:
These defaults I've taken from Tendermint; it is important to check that Tendermint will not set them, if they are not set by us.  but yes, we definitely want to look at disabling tendermints internal rpc for in-process tendermint